ui/src/components/FormField/LabelField.tsx

```diff
@@ -85,7 +85,7 @@ const LabelField: React.FC<LabelFieldProps & TextFieldProps> = ({ isKV = false,
       setError('')
     }
 
-    if (e.key === ' ') {
+    if (e.key === 'Tab') {
```

Comment:
```js
if (e.key === 'Tab') {
  e.preventDefault()

  processText()
}
```

Add `Event.preventDefault()` to prevent to lost input focus after processing text.

Reply:
fixed